cmds/core/yes/yes_test.go

```diff
@@ -0,0 +1,42 @@
+// Copyright 2017-2019 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"log"
+	"testing"
+
+	"github.com/u-root/u-root/pkg/testutil"
+)
+
+func TestMain(m *testing.M) {
+	testutil.Run(m, main)
+}
+
+func TestYes(t *testing.T) {
+	cmd := testutil.Command(t)
+	fmt.Println(cmd.Path)
+	stdout, err := cmd.StdoutPipe()
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	if err := cmd.Start(); err != nil {
+		log.Fatal(err)
+	}
+
+	in := bufio.NewScanner(stdout)
+
+	for i := 0; i < 20; i++ {
+		if in.Scan() {
+			log.Println(in.Text())
```

Comment:
don't use log. anything, use t.Logf

Reply:
i := in.Text()
if i != "y" {
log.Errorf("in.Text: got %v, want y", i)
break
}

Reply:
Fixed!